doc/src/manual/unicode-input.md

```diff
@@ -0,0 +1,66 @@
+# Unicode Input
+
+!!! warning
+
+    This table may appear to contain missing characters in the second column, or even
+    show characters that are inconsistent with the characters as they are rendered in
+    the Julia REPL. In these cases, users are strongly advised to check their choice
+    of fonts in their browser and REPL environment, as there are known issues with
+    glyphs in many fonts.
+
+```@eval
+#
+# Generate a table containing all LaTeX and Emoji tab completions available in the REPL.
+#
+
+function tab_completions(symbols...)
+    completions = Dict{String, Vector{String}}()
+    for each in symbols, (k, v) in each
+        completions[v] = push!(get!(completions, v, String[]), k)
+    end
+    return completions
+end
+
+function unicode_data()
+    file = normpath(JULIA_HOME, "..", "..", "doc", "UnicodeData.txt")
+    url = "http://www.unicode.org/Public/UCD/latest/ucd/UnicodeData.txt"
```

Comment:
Downloading this list from an external source doesn't sound like a good idea to me. That means the data can change without Julia supporting newly-introduced forms (since IIUC this depends on the utf8proc version we use). How about adding this file to git? Or couldn't we get this information from utf8proc itself?